### PR TITLE
[3.5] CHANGELOG: support multiple values for allowed client and peer …

### PR DESCRIPTION
**Pull request by lhy1024**
…TLS identities

Please read https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-3.5.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ### Dependencies
 - Compile binaries using [go 1.21.11](https://github.com/etcd-io/etcd/pull/18129).
 - [Fully address CVE-2023-45288 and fix govulncheck CI check](https://github.com/etcd-io/etcd/pull/18170)
+- Add [Support multiple values for allowed client and peer TLS identities](https://github.com/etcd-io/etcd/pull/18160)
```
Comment (jmhbnz) [inline CHANGELOG/CHANGELOG-3.5.md:12]: Please add a new `### etcd server` section for `v3.5.15` and move this entry underneath it.

**Comment by ahrtr** [comment]
Please rebase this PR, thx.

I believe there is a conflict

https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.5.md#v3515-tbd

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @lhy1024

**Review by ivanvc — APPROVED** [review]
LGTM. Thanks, @lhy1024.

**Pull request merged**